core/src/main/java/com/airbnb/aerosolve/core/models/AdditiveModel.java

```diff
@@ -23,7 +23,7 @@
 // A generalized additive model with a parametric function per feature.
 // See http://en.wikipedia.org/wiki/Generalized_additive_model
 @Slf4j
-public class AdditiveModel extends AbstractModel {
+public class AdditiveModel extends AbstractModel implements Cloneable {
```

Comment:
Add it to Model as Model deepCopy() maybe too big at this time, how bout  public AdditiveModel deepCopy() ?